### PR TITLE
CI/CD: offline_sev_kbc CI

### PR DESCRIPTION
**Pull request by dubek**
Signed-off-by: Dov Murik <dovmurik@linux.ibm.com>

----

Following #27 , add a build "job" for the offline_sev_kbc. It builds and runs the rust tests with `--feature offline_sev_kbc`. 

How can we test this?

cc: @fitzthum

### CODE REVIEW

**Comment by dubek** [comment]
I'll wait until #31 is merged, then I'll rebase. Hopefully it will solve the CI issues.

**Comment by dubek** [comment]
rebased on top of main; we can try the github workflow now.

**Review by sameo — APPROVED** [review]

**Pull request merged**
